app/src/main/res/layout/play_each_verse.xml

```diff
@@ -31,4 +31,11 @@
       app:np_fadingEdgeEnabled="true"
       app:np_wrapSelectorWheel="true"
       />
+
+  <CheckBox
+      android:id="@+id/checkBoxInfiniteVerse"
+      android:layout_width="wrap_content"
+      android:layout_height="match_parent"
+      android:layout_weight="1"
+      android:text="@string/infinity" />
```

Comment:
Although `infinity` is an attractive word, it sounds weird in some language translations. Also, we had this issue #595.
I suggest to change it back to `loop`.

Reply:
`@string/infinity` is not translatable and [is just the Unicode character](https://github.com/quran/quran_android/blob/eb557e823bd2eb48f064f19108d906aa787d4955/app/src/main/res/values/strings.xml#L290) (∞). I don't mind using actual words (loop, etc), but don't know if all languages will fit, since the checkbox is next to the horizontal number picker.

Reply:
I see, thanks. I have not noticed `infinity` was a symbol.